packages/injected-script/sdk/bridgedEthereum.ts

```diff
@@ -29,13 +68,17 @@ export const bridgedEthereumProvider: BridgedEthereumProvider = {
 }
 export interface BridgedEthereumProvider {
     /** Wait for window.ethereum object appears. */
-    untilAvailable(): Promise<void>
+    untilAvailable(): Promise<true>
     /** Send JSON RPC to the eth provider. */
     request(data: unknown): Promise<unknown>
+    /** Send JSON RPC  */
+    send(payload: JsonRpcPayload, callback: (error: Error | null, result?: JsonRpcResponse) => void): void
+    /** Async send JSON RPC  */
+    sendAsync(payload: JsonRpcPayload, callback: (error: Error | null, result?: JsonRpcResponse) => void): void
```

Comment:
send and sendAsync are deprecated in the spec. Do you really need them?

Reply:
Need finer control on payload id.